torch/testing/_internal/common_utils.py

```diff
@@ -1302,6 +1299,27 @@ def wrapper(*args, **kwargs):
 
     return decorator
 
+def markDynamoStrictTest(cls_or_func):
+    """
+    Marks the test as 'strict'. In strict mode, we reset before and after the
+    test, and run without suppress errors.
+    """
+    if inspect.isclass(cls_or_func):
+        cls_or_func.dynamo_strict = True
+        return cls_or_func
+
+    fn = cls_or_func
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        torch._dynamo.reset()
+        with unittest.mock.patch("torch._dynamo.config.suppress_errors", False):
+            fn(*args, **kwargs)
+        torch._dynamo.reset()
```

Comment:
This doesn't really work on classes, right?

Reply:
the class handling is L1307

Reply:
Oh I see, I read this too quickly. Thanks for checking.